### DISCUSSION TITLE
Should attendance number be a pulldown?

### DISCUSSION THREAD

**Discussion by mimiyin** [ITPNYU/booking-app#60] [Ideas]
Do we want individual numbers or categories?
What are the broad categories we care about?
1-10
11-30?
I can't remember what the max capacity is... but it's low.

**Comment by nopivnick** [comment]
I think this depends in part on how granular @ss12430 wants her metrics to be.

**Reply by ss12430** [reply]
I think this is a good idea. Would it be possible to make it where people cannot request a number above the capacity for each room?

**Reply by nopivnick** [reply]
sorry, which is a good idea?

if it's a dropdown, the user can only select the options we give them.

if it's a text field, we use regex to do data validation to check whether a. it's a numeral and b. whether it's less than or equal to whatever the capacity of the room is.

**Comment by ss12430** [comment]
A good idea re: Mimi's question "Should attendance number be a pulldown?"

Say someone picks 220 (capacity 30) and 233 (capacity 50). It would be very helpful if the form is smart enough to only allow someone to choose an attendance of up to 80. I don't mind whether that's a dropdown or text field.
